lib/bookmark_manager.rb

```diff
@@ -7,22 +7,27 @@ def initialize(user)
     @user = user
   end
 
-  def create(post_id:, name: nil, reminder_type: nil, reminder_at: nil, options: {})
-    post = Post.find_by(id: post_id)
+  def create(post_id:, topic_id: nil, name: nil, reminder_type: nil, reminder_at: nil, options: {})
```

Comment:
```suggestion
  def create(topic_id:, post_id: nil, name: nil, reminder_type: nil, reminder_at: nil, options: {})
```

This is seems more intuitive to me because `topic_id` is none null-able